converter/internal/promtailconvert/testdata/kubernetes.river

```diff
@@ -0,0 +1,70 @@
+discovery.kubernetes "fun_0" {
+	api_server = "http://toby.pets.com/"
+	role       = "pod"
+
+	basic_auth {
+		username      = "robin"
+		password_file = "/home/robin/.password"
+	}
+	proxy_url = "http://proxy.example.com"
+
+	tls_config {
+		ca_file              = "/home/robin/.ca"
+		cert_file            = "/home/robin/.cert"
+		key_file             = "/home/robin/.key"
+		server_name          = "example.local"
+		insecure_skip_verify = true
+	}
+}
+
+discovery.kubernetes "fun_1" {
+	role            = "pod"
+	kubeconfig_file = "/home/toby/.kube/config"
+}
+
+discovery.kubernetes "fun_2" {
+	api_server = "http://toby.pets.com/"
+	role       = "pod"
+
+	authorization {
+		type             = "Bearer"
+		credentials_file = "/home/robin/.special_token"
+	}
+}
+
+discovery.kubernetes "fun_3" {
+	api_server = "http://toby.pets.com/"
+	role       = "pod"
+
+	authorization {
+		type             = "Bearer"
+		credentials_file = "/home/toby/.token"
+	}
+}
+
+discovery.kubernetes "fun_4" {
+	api_server = "http://toby.pets.com/"
+	role       = "node"
+
+	oauth2 {
+		client_id          = "client_id"
+		client_secret_file = "foo/bar"
+		scopes             = ["scope1", "scope2"]
+		token_url          = "https://example/oauth2/token"
+		endpoint_params    = {
+			host = "example",
+			path = "/oauth2/token",
+		}
+
+		tls_config { }
+	}
+}
+
+discovery.file "fun" {
+	path_targets = concat(discovery.kubernetes.fun_0.targets, discovery.kubernetes.fun_1.targets, discovery.kubernetes.fun_2.targets, discovery.kubernetes.fun_3.targets, discovery.kubernetes.fun_4.targets)
```

Comment:
tiny nit: it would be nice to format these on separate lines for readability:

```suggestion
	path_targets = concat(
		discovery.kubernetes.fun_0.targets, 
		discovery.kubernetes.fun_1.targets, 
		discovery.kubernetes.fun_2.targets, 
		discovery.kubernetes.fun_3.targets, 
		discovery.kubernetes.fun_4.targets,
	)
```